apps/mobile/src/app/pages/page-signin/sign-in.tsx

```diff
@@ -0,0 +1,17 @@
+import { SafeAreaView, Text } from 'react-native'
```

Comment:
- [ ] - change folder name of page-signin start with capital letter without word page. -> Signin